packages/replay-internal/src/util/createPerformanceEntries.ts

```diff
@@ -191,14 +191,17 @@ export function getLargestContentfulPaint(metric: Metric): ReplayPerformanceEntr
  * Add a CLS event to the replay based on a CLS metric.
  */
 export function getCumulativeLayoutShift(metric: Metric): ReplayPerformanceEntry<WebVitalData> {
-  // get first node that shifts
-  const firstEntry = metric.entries[0] as (PerformanceEntry & { sources?: LayoutShiftAttribution[] }) | undefined;
-  const node = firstEntry
-    ? firstEntry.sources && firstEntry.sources[0]
-      ? firstEntry.sources[0].node
-      : undefined
-    : undefined;
-  return getWebVital(metric, 'cumulative-layout-shift', node);
+  const lastEntry = metric.entries[metric.entries.length - 1] as (PerformanceEntry & { sources?: LayoutShiftAttribution[] }) | undefined;
```

Comment:
Ahh, looks like this aligns with the `browserTracing` logic where we also take the last entry. Sounds good to me!

https://github.com/getsentry/sentry-javascript/blob/develop/packages/browser-utils/src/metrics/browserMetrics.ts#L219

Reply:
Is there a chance we can misalign (say we change the behavior in perf later?)
If so, is there a world we create some convinience function on a shared/core packaged used by both? or is that just not worth it?

Reply:
I'm afraid we are going to missalign a bit at least: I opened https://github.com/getsentry/sentry-javascript/pull/13056 with an experimental change and realized that it breaks how we listen to LCP for replay breadcrumbs. 

However, I think we just need to be aware that we're tracking CLS a bit differently for replay than for performance insights. In Replay we can collect CLS whenever it emits (though I guess we need to debounce that a bit to acomodate for layout shift bursts) but we can't do the same easily for insights.

Reply:
I'll for now probably decouple the listeners for `browserTracing` and replay: https://github.com/getsentry/sentry-javascript/pull/13056#issuecomment-2251093646